### PR TITLE
HW1 [Part 2,3]: Added projects to ag4219.md

### PR DESCRIPTION
**Pull request by angarc**

### CODE REVIEW

**Comment by angarc** [comment]
I think I did something I wasn't supposed to do. I saw that there was a conflict in my README from a previous commit, but it gave me the option to resolve it using the resolve tool on github, so i tried it.

But then after resolving it, ALL the <uni>.md files showed up in my hw1 branch! I'm not sure if I should close this PR and try again, or just leave it? @timkpaine

File: chessjs.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Chess.js](https://github.com/jhlywa/chess.js) is a headless chess logic library that you can use if you wanted to build a chess game in javascript, and you already had the UI ready.
```
Comment (timkpaine) [inline chessjs.md:1]: move to `projects/javascript/chessjs.md`

File: streeteasy-scraper.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+This is a webscraper idea for the website steeteasy.com, which is a real estate platform similar to zillow.com but specific to new york city.
```
Comment (timkpaine) [inline streeteasy-scraper.md:1]: move to `projects/javascript/streeteasy-scraper.md` (or whatever language you're using)

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: chessjs.md (hunk 1 above)
Comment (angarc) [reply chessjs.md]: Done.

File: streeteasy-scraper.md (hunk 2 above)
Comment (angarc) [reply streeteasy-scraper.md]: Done.

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**